### PR TITLE
What's new in transformations 10.2

### PR DESCRIPTION
**Pull request by oscarkilhed**

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -39,3 +39,40 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+
+<!-- Solomon Dubock, BI Squad -->
+
+With the new format string transformation you can manipulate string fields to look nicer! The currently supported operations are.
+
+- **Change case** changes the case of your string to upper case, lower case, sentence case, title case, pascal case, camel case or snake case.
+- **Trim** removes white space characters at the start and end of your string.
+- **Substring** selects a part of your string field.
+
+Learn more about the string format transformation [here]({{< relref="./panels-visualizations/query-transform-data/transform-data/#format-string" >}})
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:78]:
```suggestion
Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string)
```

**Review by imatwawana — COMMENTED** [review]
Made some style and wording edits and left some questions about feature availability.

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -39,3 +39,40 @@ You must use relative references when linking to docs within the Grafana repo. P
...
+
+As our work on improving the user experience of transforming data continues, we've also been adding new capabilities to transformations.
+
+### Support for dashboard variables in transformations
+
+<!-- Oscar Kilhed, Victor Marin -->
+
+_Experimental in all editions of Grafana_
+
+Previously the only transformation that supported [dashboard variables]({{< relref "../dashboards/variables/" >}}) was the **add field from calculation transformation**. We have now extended the support for variables to **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name** and **Join by field**. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support variables, either in the dropdown or as a suggestion when you type **$** or press `<ctrl+space>` {{< figure src="/media/docs/grafana/transformations/completion.png" caption="Input with dashboard variable suggestions" >}}
```
[the remark this answers is not in the document]
Comment (oscarkilhed) [reply docs/sources/whatsnew/whats-new-in-v10-2.md:53]: thank you

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 3]
```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+- **Absolute value (abs)** - Returns the absolute value of a given expression. It represents its distance from zero as a positive number.
+- **Natural exponential (exp)** - Returns _e_ raised to the power of a given expression.
+- **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
+- **Floor (floor)** - Returns the largest integer less than or equal to a given expression.
+- **Ceiling (ceil)** - Returns the smallest integer greater than or equal to a given expression.
+
+{{< figure src="/media/docs/grafana/transformations/unary-operation.png" >}}
+
+**Row index** adds a field that represents the row index of the row.
```
Comment (mdvictor) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:103]: I think this needs modification. For this release we only added a new switch `As percentage` to show the existing row index as a percent instead. I can make the modifications here

Comment (mdvictor) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: @oscarkilhed let me know what you think of the changes

Comment (oscarkilhed) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: Ah, my mistake, thanks for the correction @mdvictor

Comment (mdvictor) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: no worries!

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 4]
```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+_Experimental in Grafana Open Source and Enterprise_
+
+With the new **Format string** transformation, you can manipulate string fields to improve how they're displayed. The currently supported operations are:
+
+- **Change case** changes the case of your string to upper case, lower case, sentence case, title case, pascal case, camel case, or snake case.
+- **Trim** removes white space characters at the start and end of your string.
+- **Substring** selects a part of your string field.
+
+Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string)
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:119]:
```suggestion
Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string)
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 5]
```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+- **Natural exponential (exp)** - Returns _e_ raised to the power of a given expression.
+- **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
+- **Floor (floor)** - Returns the largest integer less than or equal to a given expression.
+- **Ceiling (ceil)** - Returns the smallest integer greater than or equal to a given expression.
+
+{{< figure src="/media/docs/grafana/transformations/unary-operation.png" >}}
+
+**Row index** can now show the index as a percentage.
+
+Learn more in the [Add field from calculation documentation](https://grafana.com/docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation)
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:105]:
```suggestion
Learn more in the [Add field from calculation documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation)
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 6]
```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+As our work on improving the user experience of transforming data continues, we've also been adding new capabilities to transformations.
+
+### Support for dashboard variables in transformations
+
+<!-- Oscar Kilhed, Victor Marin -->
+
+_Experimental in Grafana Open Source and Enterprise_
+
+Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:81]:
```suggestion
Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 7]
```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+- **Natural exponential (exp)** - Returns _e_ raised to the power of a given expression.
+- **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
+- **Floor (floor)** - Returns the largest integer less than or equal to a given expression.
+- **Ceiling (ceil)** - Returns the smallest integer greater than or equal to a given expression.
+
+{{< figure src="/media/docs/grafana/transformations/unary-operation.png" >}}
+
+**Row index** can now show the index as a percentage.
+
+Learn more in the [Add field from calculation documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation)
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:105]:
```suggestion
Learn more in the [Add field from calculation documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation).
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 8]
```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+_Experimental in Grafana Open Source and Enterprise_
+
+With the new **Format string** transformation, you can manipulate string fields to improve how they're displayed. The currently supported operations are:
+
+- **Change case** changes the case of your string to upper case, lower case, sentence case, title case, pascal case, camel case, or snake case.
+- **Trim** removes white space characters at the start and end of your string.
+- **Substring** selects a part of your string field.
+
+Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string)
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:119]:
```suggestion
Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).
```

**Review by imatwawana — APPROVED** [review]
A couple small nits, but otherwise this LGTM!

**Pull request merged**